unit-tests/src/test/scala/java/util/FormatterSuite.scala

```diff
@@ -2492,7 +2487,7 @@ object FormatterSuite extends tests.Suite {
 
   testFails(
     "format(String, Array[Object]) for Float/Double conversion type 'e' and 'E'",
-    0) { // issue not filed yet
+    1296) {
```

Comment:
Why does this one still fail? Is it going to be easy to fix after this PR?

Reply:
@denesh You have sharp eyes!

You are entirely right, the continuing failure is enough to astonish a reviewer or
future maintainer.

It took me a while to warp my brain again to understand the continuing failure.
I created new issue #1443 to describe the continuing failure.

The short story is that after this PR or its successors are accepted, the test
will continue to fail because it will then get as far as a LOCALE.German test
at the bottom of the file and LOCALE.German is not currently implemented.

The original SN developer created the file FormatterUSSuite.scala to avoid
Locale issues. The numeric portions for this test are, to quick inspection,
identical and pass (after this PR or kin).

Reply:
We are most likely never going to support non-US locales, so might as well drop tests which depend on those locales being implemented altogether.

Reply:
Yes `Locale` and friends are bigger than a breadbox. 

You will probably be asking me to write at least the first
draft of a release note. I should put in a prominent
place that the current .toString() code always uses dor or
full-stop for the decimal place and never uses a places separator.
It is not US Local and I would have to check if it even matches C Locale.
Almost certainly better than existing printf() based code, but not 
an exact JVM match.

The ideas of 'fast & light binaries ' and of binaries which can
be easily used by all those in the non-English world are probably
diametrically opposed.  I suspect that it will come down to a 
two-library choice at link time or such.   
 
FormatterUSSuite.scala has us-specific tests for this PR.

This commit puts an explanatory issue number on each of the
testFails() relevant to it.

I propose  that I spend my time on removing BigInteger rather
than stripping stuff out of FormatterSuite.scala. That file is no worse
than it was and is even a little better with issue markers.

I'll keep my eye out to see if Locale stuff can be stripped from 
Ryu code.

Please advise.

PS: In have dreams of taking a run at Locale after the PRs I have currently outstanding
       have been dispatched.  No promises, just dreams & hopes.

Reply:
I suggest dropping german locale tests completely. If we ever support non-US locales, we can bring it back then (and it's extremely unlikely).

Reply:
I will drop the Locale test in the specific test as you suggest.

If you are are recommending that all Locale tests be dropped from
FormatterSuite.scala then it would be less astonishing and more
well contained if I did that in a second PR, subsequent to this one
being accepted.  The changes are more extensive. In addition 
FormatterUSSuite.scala becomes redundant and should be deleted.
Someone looking to revert the larger changes are unlikely to find them
in this PR.

Your thoughts/direction?  Thank you.

Reply:
Doing it in a separate PR sounds good.

Reply:
Understood & agreed, in a subsequent, independent PR nuke all Locale references
in FormatterSuite.scala and mark FormatterUSSuite.scala as obsolete.
I will also close the corresponding issue.

I think the changes to FormatterSuite.scala in this PR are worth doing as part of
this PR.  They give a marker for what needs to change.  I could figure out the
git foo to revert that file is you want.